sdk/src/interfaces.ts

```diff
@@ -88,20 +93,22 @@ export interface ProjectOptions {
    */
   hideDevTools?: boolean;
   /**
-   * Hide the sidebar on page load.
-   *
-   * Users will still be able to open the sidebar by clicking one of the sidebar icons.
+   * Hide the ActivityBar (sidebar icons).
```

Comment:
Tentatively fixed the description for the `hideExplorer` option.

It's hard to describe in non-jargony terms, and I'm guessing we added this arcane feature for some customer at one point that didn't want the sidebar or ActivityBar to show at all. This may become obsolete if we change the embed layout to use a drawer menu like the main layout does on small screen (either all the time in embeds, or only on small screens).